src/panels/lovelace/editor/card-editor/hui-card-picker.ts

```diff
@@ -265,18 +284,22 @@ export class HuiCardPicker extends LitElement {
         >
           ${!element || element.tagName === "HUI-ERROR-CARD"
             ? html`
-                ${this.hass!.localize(
-                  `ui.panel.lovelace.editor.card.${cardConfig.type}.description`
-                )}
+                ${customCard && customCard.description
+                  ? customCard.description
+                  : this.hass!.localize(
+                      `ui.panel.lovelace.editor.card.${cardConfig.type}.description`
+                    )}
```

Comment:
```suggestion
                ${customCard?.description
                  || this.hass!.localize(
                      `ui.panel.lovelace.editor.card.${cardConfig.type}.description`
                    )}
```

Reply:
Same. Localize will return empty if it's a custom card without description. Expected ?